.github/workflows/check.yml

```diff
@@ -20,3 +20,7 @@ jobs:
         with:
           command: check
           args: --workspace --examples
+      - uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --doc --target x86_64-unknown-linux-gnu
```

Comment:
Since this is a test workflow, and it's targeting the host, could you add it to
```.github/workflows/tests_host.yml```
instead of
```.github/workflows/check.yml```

Reply:
Are you sure? I changed all examples to no_run since we can only compile check them. So I feel  this is basically a check for them. I interpreted tests_host as relevant for running actual tests on either iron or the ci host (with mocked hardware?)

Reply:
You are correct, they do logically fit with the checks so let's leave them there